src/main/java/net/shadowmage/ancientwarfare/npc/config/AWNPCStatics.java

```diff
@@ -157,7 +157,7 @@ public int getFoodValue(ItemStack stack) {
 		if (foodValues.containsKey(name)) {
 			return foodValues.get(name);
 		} else if (stack.getItem() instanceof ItemFood) {
-			return (((ItemFood) stack.getItem()).getHealAmount(stack) + (int)((ItemFood) stack.getItem()).getSaturationModifier(stack)) * foodMultiplier;
+			return ((((ItemFood) stack.getItem()).getHealAmount(stack)) + ((int)((((ItemFood) stack.getItem()).getSaturationModifier(stack))) * 10)) * foodMultiplier;

```

Comment:
I have just checked how vanilla adds saturation to the player when you eat a food and this is the calculation there - healAmount * foodSaturationModifier * 2.0F so we should use the same